### PR TITLE
chore: edit values file for new server options specification

### PR DESCRIPTION
**Pull request by olevski**
Depends on https://github.com/SwissDataScienceCenter/renku-notebooks/pull/649 and should be merged together with https://github.com/SwissDataScienceCenter/renku-notebooks/pull/649

### CODE REVIEW

**Review by aledegano — APPROVED** [review]

File: helm-chart/values.yaml.changelog.md [hunk 1]
```diff
@@ -6,7 +6,20 @@ Please follow this convention when adding a new row
 * `<type: NEW|EDIT|DELETE> - *<resource name>*: <details>`
 
 ----
-## Unreleased (breaking changes)
+
+## Upgrading to Renku 0.8.x (Unreleased)
+* NEW *notebooks.serverDefaults* has been added with default values that will be
```
Comment (ableuler) [inline helm-chart/values.yaml.changelog.md:11]: I'd try to keep this short and just link to the new section in the chart readme that you also add: https://github.com/SwissDataScienceCenter/renku/tree/master/helm-chart#upgrading-to-08x (I know it's a bit annoying to make those links ahead of time for unreleased tags)

**Review by ableuler — CHANGES_REQUESTED** [review]
Good description of the new value, I'd just try to avoid duplication.

**Comment by rokroskar** [comment]
@olevski @ableuler does this need to go in asap?

**Comment by ableuler** [comment]
> @olevski @ableuler does this need to go in asap?

No. It should be merged once we do the next release in the notebooks-service and bump the corresponding chart requirement.

**Pull request closed without merging**